src/js/helpers/SecretsUtil.js

```diff
@@ -0,0 +1,26 @@
+var SecretsUtil = {
+  environmentVariableValueWithSecret: function (value, allFields) {
```

Comment:
Please rename this method to better describe what it is doing as well as the arguments to reflect their type.